eslint.config.mjs

```diff
@@ -62,7 +62,7 @@ export default [
         }
     },
     {
-        files: ["po/po2js", "po/xgettext-html"],
+        files: ["po/po2js.js", "po/xgettext-html.js"],
```

Comment:
There doesn't seem to be any need to add any extension to these commands now?

Reply:
No need, can change back